lisa/transformers/kernel_source_installer.py

```diff
@@ -77,6 +76,20 @@ class SourceInstallerSchema(BaseInstallerSchema):
     # Steps to modify code by patches and others.
     modifier: List[BaseModifierSchema] = field(default_factory=list)
 
+    # This is absolute path where kernel config file is located
+    kernel_config_file: str = field(
+        default="",
+        metadata=field_metadata(
+            required=False,
+        ),
+    )
+
+
+@dataclass_json()
+@dataclass
+class Dom0InstallerSchema(SourceInstallerSchema):
```

Comment:
Because there is no new thing from schema. it doesn't need to declare a schema for dom0.

Reply:
This is to maintain installer and its corresponding schema

Reply:
If extra parameter needed, it can be added in future. It doesn't need a new schema for each new type. The Installer can reference the source installer schema.

Reply:
ok chi i will remove dom0 schema and refer to source schema in dom0 installer

Reply:
Done